components/IndexPage/Content.js

```diff
@@ -172,7 +198,16 @@ export const IndexContent = withTranslation("index")(
           />
         </Text>
 
-        <ChipBadge color="success" label="Updated" icon="check-circle" />
+        {childId === "0" && (
+          <ChipBadge color="success" label="Updated" icon="check-circle" />
+        )}
+
+        {childId !== "0" && (
+          <Row>
+            <ChipBadge color="error" label="Outdated" />
```

Comment:
@magiiixam1997 I tried to pass here exclamation icon but it doesn't work. How we can use other than `check-circle` icons?